handlers/agentinfo/info.go

```diff
@@ -65,13 +67,14 @@ type agentInfo struct {
 }
 
 type agentStats struct {
-	CpuPercent          *float64 `json:"cpu,omitempty"`
-	MemoryBytes         *uint64  `json:"mem,omitempty"`
-	FileDescriptorCount *int32   `json:"fd,omitempty"`
-	ThreadCount         *int32   `json:"th,omitempty"`
-	LatencyMillis       *int64   `json:"lat,omitempty"`
-	PayloadBytes        *int     `json:"load,omitempty"`
-	StatusCode          *int     `json:"code,omitempty"`
+	CpuPercent           *float64 `json:"cpu,omitempty"`
+	MemoryBytes          *uint64  `json:"mem,omitempty"`
+	FileDescriptorCount  *int32   `json:"fd,omitempty"`
+	ThreadCount          *int32   `json:"th,omitempty"`
+	LatencyMillis        *int64   `json:"lat,omitempty"`
+	PayloadBytes         *int     `json:"load,omitempty"`
+	StatusCode           *int     `json:"code,omitempty"`
+	SharedConfigFallback *int     `json:"scfb,omitempty"`
```

Comment:
nit: `scfb` isn't intuitive to understand. Maybe something like `cred_fallback`? 

Don't feel too strong on this since it looks like only the developers will use this.

Reply:
We have a limit on the header size, so we're trying to keep the names brief. It's an acronym (kinda).